docs/pages/guides/docs.md

```diff
@@ -29,7 +29,7 @@ and uses the modern MyST plugin to get Markdown support.
 >   based on markdown and HTML. Less support for man pages & PDFs than Sphinx,
 >   since it doesn't use docutils. Has over
 >   [200 plugins](https://github.com/mkdocs/catalog) - they are much easier to
->   write than Sphinx. Examples include [hatch](https://hatch.pypa.io),
+>   write than Sphinx. Example sites include [hatch](https://hatch.pypa.io),
```

Comment:
clarification: these are example sites not example plugins; context makes that not super clear.